### PR TITLE
docs(docs-infra): enable tslint

### PR DESCRIPTION
**Pull request by JeanMeche**

### CODE REVIEW

File: adev/shared-docs/pipeline/guides/testing/docs-code/docs-code.spec.ts [hunk 1]
```diff
@@ -13,6 +21,7 @@ describe('markdown to html', () => {
       runfiles.resolvePackageRelative('docs-code/docs-code.md'),
       {encoding: 'utf-8'},
     );
+    console.log(await parseMarkdown(markdownContent, {}));
```
Comment (josephperrott) [inline adev/shared-docs/pipeline/guides/testing/docs-code/docs-code.spec.ts:24]: This should be able to be removed now?

File: adev/src/app/editor/code-editor/extensions/tooltip.ts [hunk 2]
```diff
@@ -10,7 +10,15 @@ import {Signal} from '@angular/core';
 import {Tooltip, hoverTooltip} from '@codemirror/view';
 import {marked} from 'marked';
 import {Subject, filter, take} from 'rxjs';
-import type {JSDocTagInfo, SymbolDisplayPart} from 'typescript';
+/**
+ * @license
```
Comment (josephperrott) [inline adev/src/app/editor/code-editor/extensions/tooltip.ts:14]: We shouldn't need a second copyright block in this file correct?

File: adev/src/app/features/references/api-items-section/api-items-section.component.spec.ts [hunk 3]
```diff
@@ -64,7 +64,9 @@ describe('ApiItemsSection', () => {
     );
     const deprecatedApiTitle = deprecatedApiIcons[0].parent?.query(By.css('.adev-item-title'));
 
+    //tslint:disable-next-line:no-console
     console.log('DEPRECATED API ICONS', deprecatedApiIcons);
```
Comment (josephperrott) [inline adev/src/app/features/references/api-items-section/api-items-section.component.spec.ts:68]: I don't think we should be logging these to the console during a test like this anyway right? We should be able to just remove the log lines?

**Review by josephperrott — CHANGES_REQUESTED** [review]

**Review by josephperrott — APPROVED** [review]
LGTM

**Comment by alxhub** [comment]
This PR was merged into the repository by commit f15ccb94b9737741e66f00af32f282a29b06f7a0.

The changes were merged into the following branches: main, 19.0.x

**Pull request closed without merging**